README.md

```diff
@@ -31,7 +33,175 @@ $ pip install git+ssh://git@github.com/wizeline/relayer.git
 
 ## Usage
 
-TODO
+Relayer supports two different clients, RPC and Flask, depending on which client do you need the initialization steps differ, but once you have a `relayer` object the usage is identical, the relayer object provides two methods:
+
+`relayer.emit(event_type, event_subtype, payload, partition_key=None)`: This method writes directly to a kafka topic, the kafka topic is defined by the `event_type` argument, if `partition_key` is provided all messages sent to that specific `partition_key` are guaranteed to get written to the same partition and the `key` of the message will be the binary representation of that key, the partition key can only be a string, a bytes string or a uuid. The message written is the json encoding of a dictionary containing `event_type`, `event_subtype` and `payload` keys, if the message is not json serializable the exception `NonJSONSerializableMessageError` will be raised.
+
+`relayer.log(log_level, message)`: This method logs a message, all the messages logged gets written to a specific topic at the end of the request (or RPC call), more on this later.
+
+When you do a `relayer.emit` the message gets sent immediatly to kafka, but also a copy gets saved for the purpose of log aggregation. When you create the relayer a `logging_topic` needs to be provided, this argument represents a kafka topic where log information gets written, all the messages written by `relayer.emit` and `relayer.log` gets aggregated for every request (or RPC call) and gets written to the specified `logging_topic` on kafka, so it can later be piped to an ELK stack or similar systems. This special log message also contains metadata related to the request, such as the timestamp and how long did it take to process the request.
+
+
+### Flask
+
+The Flask relay is a Flask extension, it works by wrapping your Flask application in a middleware so it can aggregate al events and logs emitted by request and flush them out at the end of the request.
+
+__How to use:__
+
+You first need to create the FlaskRelayer extension inside your `extensions.py`, for instance:
+
+```python
+from relayer.flask import FlaskRelayer
+
+
+relayer = FlaskRelayer()
+```
+
+After creating it you need to initialize it on your `app.py`
+
+```python
+from flask import Flask
+
+from .extensions import relayer
+
+def create_app(config_object=None):
+    app = Flask(__name__)
+    app.config.from_object(config_object)
+    register_extensions(app)
+    register_blueprints(app)
+    register_errorhandlers(app)
+    return app
+
+def register_extensions(app):
+    relayer.init_app(app, 'logging_topic', kafka_hosts='kafka')
+```
+
+Once you've done that you can import `extensions.relayer` from anywhere on your Flask application and use `relayer.emit` and `relayer.log`
```

Comment:
I know it's trivial but it would be nice to add a codeblock with the example of importing and using these two methods.